imports/plugins/core/accounts/client/containers/updatePassword.js

```diff
@@ -83,7 +86,7 @@ const wrapComponent = (Comp) => (
           } else if (!storefrontUrls || !storefrontUrls.storefrontLoginUrl) {
             throw new ReactionError("error-occurred", "Missing storefront URLs. Please set these properties from the shop settings panel.");
           } else {
-            window.location.href = storefrontUrls.storefrontLoginUrl;
+            Router.go(storefrontUrls.storefrontLoginUrl);
```

Comment:
This should still probably be `window.location.href` you're redirecting to a different site. Router.go is really only for push state within this app context.